scipy/stats/_stats_py.py

```diff
@@ -5551,6 +5563,159 @@ def _mgc_stat(distx, disty):
+        two-sample MGC test will be run.
+    compute_distance : callable, optional
+        A function that computes the distance or similarity among the samples
+        within each data matrix. Set to ``None`` if ``x`` and ``y`` are
+        already distance matrices. The default uses the euclidean norm metric.
+        If you are calling a custom function, either create the distance
+        matrix before-hand or create a function of the form
+        ``compute_distance(x)`` where `x` is the data matrix for which
+        pairwise distances are calculated.
+    reps : int, optional
+        The number of replications used to estimate the null when using the
+        permutation test. The default is ``1000``.
+    workers : int or map-like callable, optional
+        If ``workers`` is an int the population is subdivided into ``workers``
+        sections and evaluated in parallel (uses ``multiprocessing.Pool
+        <multiprocessing>``). Supply ``-1`` to use all cores available to the
+        Process. Alternatively supply a map-like callable, such as
+        ``multiprocessing.Pool.map`` for evaluating the p-value in parallel.
+        This evaluation is carried out as ``workers(func, iterable)``.
+        Requires that `func` be pickleable. The default is ``1``.
+    is_twosamp : bool, optional
+        If `True`, a two sample test will be run. If ``x`` and ``y`` have
+        shapes ``(n, p)`` and ``(m, p)``, this optional will be overriden and
+        set to ``True``. Set to ``True`` if ``x`` and ``y`` both have shapes
+        ``(n, p)`` and a two sample test is desired. The default is ``False``.
+        Note that this will not run if inputs are distance matrices.
+    random_state : {None, int, `numpy.random.Generator`,
+                    `numpy.random.RandomState`}, optional
+
+        If `seed` is None (or `np.random`), the `numpy.random.RandomState`
+        singleton is used.
+        If `seed` is an int, a new ``RandomState`` instance is used,
+        seeded with `seed`.
+        If `seed` is already a ``Generator`` or ``RandomState`` instance then
+        that instance is used.
+
+    Returns
+    -------
+    stat : float
+        The sample Dcorr test statistic within `[-1, 1]`.
+    pvalue : float
+        The p-value obtained via permutation.
+
+    
+    Notes
+    -----
+
+    #. Let x and y be (n,p) samples of random variables X and Y. Let D^x be the n×n 
+       distance matrix of x and D^y be the n×n be the distance matrix of y. 
+       The distance covariance is
+    
+    .. math::
+
+    Dcovnb(x,y) = (1/(n^2)) \ times trace(D^{y} H D^{y} H)
+
+    #. where tr(⋅) is the trace operator and H is defined as H=I−(1/n)J where I 
+       is the identity matrix and J is a matrix of ones. The normalized version 
+       of this covariance is distance correlation 1 and is
+    
+    .. math::
+
+    Dcorrnb(x,y) = {Dcovnb(x,y)}/{\sqrt{Dcovnb(x,x) Dcovnb(y,y)}}
```

Comment:
This LaTeX will render weird. The "n" should be a subscript. The "b" should be a superscript

Reply:
Also tab properly